docs/source/api.rst

```diff
@@ -260,6 +272,11 @@ ARIMA
 .. autoclass:: cuml.tsa.ARIMAModel
     :members:
 
+Stationarity
+------------
+
+.. autoclass:: cuml.tsa.stationarity
```

Comment:
I'm not sure this is very much an end user targetted function, @JohnZed @Nyrio  thoughts?

Reply:
No, not really. It will be useful when adding Auto-ARIMA and I will probably do a few interface changes in it then. Or maybe my implementation will use directly the C++ version.
In any case, I think it's not necessary to add it to the documentation.

Reply:
Removed it from the list